src/main/java/org/jenkinsci/plugins/fluentd/FluentHelper.java

```diff
@@ -25,7 +25,7 @@
      * @param jsonFromExtension json that would be used as extension
      * @param jsonFromFile json that would be extend
      * @param startTimeInMillis job start time in milliseconds
-     * @throws IllegalArgumentException
+     * @throws IllegalArgumentException if so.
```

Comment:
I don't believe this wording change is a good one as-is. You should explain *what* the "so" is, i.e. the exact condition under which an `IllegalArgumentException` is thrown.